application/holder/src/service/cli/open.ts

```diff
@@ -34,7 +34,7 @@ export class Action extends CLIAction {
         return this.error;
     }
 
-    public execute(cli: Service): Promise<void> {
+    public async execute(cli: Service): Promise<void> {
```

Comment:
I don't remember (please check it), but if `fucntion` is async, we can return just `Error`, instead `Promise.reject(Error)`... but please check it.

Reply:
Here is a good example to better understand promises and `async` stuff. 
```
async function a() {
    return new Error('Test');
}

async function b() {
    return Promise.reject(new Error('Test'));
}

async function c() {
    throw new Error('Test');
}

async function d() {
    return new Promise(() => {
        throw new Error('Test');
    })
}

a().then((_) => {
    console.error(`[A]: CATCH: isn't handled`);
}).catch((_) => {
    console.log('[A] - ok');
});

b().then((_) => {
    console.error(`[B]: CATCH: isn't handled`);
}).catch((_) => {
    console.log('[B] - ok');
});

c().then((_) => {
    console.error(`[C]: CATCH: isn't handled`);
}).catch((_) => {
    console.log('[C] - ok');
});

d().then((_) => {
    console.error(`[D]: CATCH: isn't handled`);
}).catch((_) => {
    console.log('[D] - ok');
});
```

console:
```
[A]: CATCH: isn't handled
[C] - ok
[B] - ok
[D] - ok
```
Well returning `new Error` will not reject promise, but return result with `resolve`